packages/lesswrong/components/localGroups/LocalGroupMarker.tsx

```diff
@@ -16,7 +16,7 @@ const styles = createStyles((theme: ThemeType): JssStyles => ({
   eaIcon: {
     width: 20,
     height: 20,
-    fill: '#90b733',
+    fill: '#4b8e10',
```

Comment:
I feel like there is probably a better color for this icon, either a different green or a totally separate color. I believe it's contributing to the icon being hard to distinguish on the map. Do we have other colors we use, or do you have a better suggestion?

Reply:
I don't have a lot of feelings here. It seems basically fine. I agree some look a bit cluttered, as they're superimposed without consideration for the city names behind them (see Geneva):

![image](https://user-images.githubusercontent.com/10352319/138072646-7cb8c2c2-0860-4087-9622-65cb62ab9481.png)

Perhaps you could apply a white dropshadow halo?